CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.11.0
+
+## Breaking Changes
+
+- #*** the async_nats crate has been merged into the
+  main nats crate, under the `asynk` module name.
```

Comment:
Can in not be called async?

Reply:
async is a reserved keyword in Rust, and this is also the name we used in the past when the async functionality was in the same crate